docs/bigtable-client-intro.rst

```diff
@@ -0,0 +1,89 @@
+Base for Everything
+===================
+
+To use the API, the :class:`Client <gcloud.bigtable.client.Client>`
+class defines a high-level interface which handles authorization
+and creating other objects:
+
+.. code:: python
+
+    from gcloud.bigtable.client import Client
+    client = Client()
+
+Long-lived Defaults
+-------------------
+
+When creating a :class:`Client <gcloud.bigtable.client.Client>`, the
+``user_agent`` and ``timeout_seconds`` arguments have sensible
+defaults
+(:data:`DEFAULT_USER_AGENT <gcloud.bigtable.client.DEFAULT_USER_AGENT>` and
+:data:`DEFAULT_TIMEOUT_SECONDS <gcloud.bigtable.client.DEFAULT_TIMEOUT_SECONDS>`).
+However, you may over-ride them and these will be used throughout all API
+requests made with the ``client`` you create.
+
+Authorization
+-------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
```

Comment:
s/should/can

We should make it more clear that you can explicitly pass the project, or it can be inferred from the environment.